<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SuperCalculator - 万能计算机</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>表达式 f(x,y,z):</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="exprEdit">
        <property name="text">
         <string>sin(x) + cos(y) * z^2</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>变量值 (x,y,z):</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_x">
          <property name="text">
           <string>x=</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="xValEdit">
          <property name="text">
           <string>1.0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_y">
          <property name="text">
           <string>y=</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="yValEdit">
          <property name="text">
           <string>0.0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_z">
          <property name="text">
           <string>z=</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="zValEdit">
          <property name="text">
           <string>2.0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>操作面板</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QPushButton" name="evalBtn">
         <property name="text">
          <string>求值 (Eval)</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="symDiffBtn">
         <property name="text">
          <string>符号求导 (d/dx)</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="numDiffBtn">
         <property name="text">
          <string>数值导数</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="intBtn">
         <property name="text">
          <string>定积分 (x从0到pi)</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="rootBtn">
         <property name="text">
          <string>求根 (f(x)=0)</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="minBtn">
         <property name="text">
          <string>求最小值</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="odeBtn">
         <property name="text">
          <string>解ODE (y'=f(x,y))</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="csvBtn">
         <property name="text">
          <string>导出数据 (CSV)</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="outputText">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>计算结果将显示在这里...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>




